### DISCUSSION TITLE
Allow node-version datasource custom registry support

### DISCUSSION THREAD

**Discussion by rwallen** [renovatebot/renovate#28046] [Suggest an Idea]
### Tell us more.

Hello,
I'm working in a self-hosted renovate environment, where the build pipeline has been closed to the open internet for security. Only designated proxies are allowed for checking dependencies. We have a proxy for `nodejs.org`, but I do not think we can configure our renovate to use it because `customRegistrySupport` has been set to `false`.

```
WARN: Custom registries are not allowed for this datasource and will be ignored
```

I was wondering if you might consider setting `customRegistrySupport` to `true` 🙇 https://github.com/renovatebot/renovate/blob/fcffdc90d82a1ce48b8a994aff29aaebe465fa50/lib/modules/datasource/node-version/index.ts#L16.

**Comment by rarkins** [comment]
I think this restriction can be removed

**Reply by rwallen** [reply]
Hey rarkins,
Apologies, I'm not sure I am able to sign the CLA 🙇.
Just wondering on what the next steps would be? Should I create an issue from the discussion as a new feat?

**Reply by rarkins** [reply]
Big companies who take from open source while not letting their employees contribute back even a false->true change are the worst

**Reply by rarkins** [reply]
But on the other hand we don't need a CLA signing for such a small change, so if you're allowed to submit a PR without signing the CLA then I can admin merge it

**Reply by rwallen** [reply]
Hey rarkins,
Apologies, I checked our policies and I should be able to sign the CLA 👍
